spec/rubocop/cop/rails/inverse_of_spec.rb

```diff
@@ -86,7 +86,7 @@ class Person
     context 'Rails >= 5.2', :config do
       let(:rails_version) { 5.2 }
 
-      it 'does not register an offense when not specifying `:inverse_of`' do
+      it 'does not register an offense when specifying `:inverse_of`' do
```

Comment:
The code in the example does not specify `inverse_of`. Why should the description be changed to say it does? 🤔